app/uk/gov/hmrc/minorentityidentificationfrontend/forms/utils/MessagesHelper.scala

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
I think this object should be in the package utils rather than forms/utils.

Reply:
done